package.json

```diff
@@ -32,51 +32,50 @@
     "predeploy": "npm run build:lib && npm run build:deploy",
     "deploy": "angular-cli-ghpages --dir ./dist/ngx-ui --name=Swimlane --email=developer@swimlane.com",
     "prepublish:lib": "npm run package",
-    "publish:lib": "npm publish ./dist/swimlane/ngx-ui",
+    "publish:lib": "npm publish ./dist/swimlane/ngx-ui --ignore-scripts",
```

Comment:
Publishing Angular library with --ignore-scripts to address NG0200 error

This npm publish command is configured with the --ignore-scripts flag to prevent the execution of post-publish scripts. The reason for this is to work around the NG0200 error (Circular dependency in DI) that occurs when compilationMode is set to 'partial' in Angular library compilation. Due to restrictions on publishing with 'full' compilationMode, --ignore-scripts is used to bypass these restrictions and successfully publish the library.